apps/create-fuels-counter-guide/src/pages/index.tsx

```diff
@@ -9,8 +9,16 @@ import { Button } from "@/components/Button";
 import toast from "react-hot-toast";
 import { useActiveWallet } from "@/hooks/useActiveWallet";
 import useAsync from "react-use/lib/useAsync";
+import { CURRENT_ENVIRONMENT } from '@/lib'
+
+// #region deploying-dapp-to-testnet-frontend-contract-id
+const contractId =
+  CURRENT_ENVIRONMENT === "local"
+    ? contractIds.testContract
+    // Replace the below string with the contract ID you copied earlier
```

Comment:
This would be an env var as mentioned

Reply:
https://github.com/FuelLabs/fuels-ts/pull/2052/commits/344fc22adbbf7dcdf9aee9d045b500816223c823